test/source/api/core/get_transactions_to_approve_test.cpp

```diff
@@ -42,7 +42,7 @@ TEST(Core, GetTransactionsToApprove) {
 
 TEST(Core, GetTransactionsToApproveWithReference) {
   IOTA::API::Core api(get_proxy_host(), get_proxy_port());
-  auto            res = api.getTransactionsToApprove(27, BUNDLE_1_TRX_1_HASH);
+  auto            res = api.getTransactionsToApprove(3, "");
```

Comment:
Why has this been removed ?

Reply:
Already explained on a private chat, but putting it here for anyone interested.

On IRI 1.5.0, there are additional checks that will reject the call to getTransactionsToApprove if the specified hash is linked to a transaction too old.

Thus, we either need to regenerate transactions before we run the tests or pass empty string here.
For now, it is probably more preferable to stick with the latter as the first one will imply lots of work and lead our testing environment to be unstable.